examples/aws-sso/Earthfile

```diff
@@ -0,0 +1,85 @@
+VERSION 0.7
+FROM amazon/aws-cli
+ARG --global EARTHLY_CI
+
+target:
+  BUILD +run --sso_region="us-east-2" --region="us-east-2" --args="sts get-caller-identity"
+
+python:
+  FROM python:3
+  WORKDIR /workspace
+
+ubuntu:
+  FROM ubuntu:jammy
+  WORKDIR /workspace
+  RUN apt update -y
+  RUN apt install -y curl unzip
+
+login:
+  IF [ "$EARTHLY_CI" = "false" ]
+    ARG --required sso_region
+    FROM +python
+    # https://github.com/benkehoe/aws-export-credentials
+    RUN pip install aws-export-credentials
+    BUILD +dev.login.open --sso_region=$sso_region
+    COPY +dev.login/aws /root/.aws
+    RUN aws-export-credentials --profile default --credentials-file-profile default
+    SAVE ARTIFACT /root/.aws/credentials /credentials
+  ELSE
```

Comment:
Should this validate `EARTHLY_CI` to be either `true` or `false`
```
ELSE IF [ "$EARTHLY_CI" = "true" ]

ELSE
  RUN echo "bad value for EARTHLY_CI" && exit 1
END
```

Reply:
My assumption was that nobody would call this by setting `EARTHLY_CI` and would rather be using the `--ci` flag, which would mean `EARTHLY_CI` is always `true` or `false`. I can add explicit checking if you think it's useful.